components/ui/LeadDetailsButton/LeadDetailsButton.tsx

```diff
@@ -0,0 +1,19 @@
+"use client";
+import Image from 'next/image';
+import React from 'react'
+import moreImage from '/assets/images/leadslist-icons/more_vert.png';
+import LeadDetails from '../LeadDetails/lead-details';
+
+const LeadDetailsButton = () => {
+  const [showModal, setShowModal] = React.useState(false);
```

Comment:
Import the `useState` into this file and use that instead of `React.useState()`

Reply:
Done